CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 # Contributing to OpenSauced Guestbook
 
+This Contributing Guide will walk you through how to contribute to this project for bug fixes and features. If you want to contribute to the guestbook, please refer to [chapter five](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source) of the Intro to Open Source course.
```

Comment:
Wasn't 100% sure if it goes here or should be on very top?

Reply:
It's already good. 👍